xtp/src/pyxtp/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+file(GLOB_RECURSE BIND_SOURCES binds/*.cc)
+
+pybind11_add_module(xtp_binds ${BIND_SOURCES})
+target_include_directories(xtp_binds PRIVATE $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/binds/include>)
+
+target_link_libraries(xtp_binds PUBLIC VOTCA::votca_xtp)
+
+if(NOT PYTHON_INSTDIR)
+  execute_process(
+    COMMAND
+      ${Python_EXECUTABLE} -c
+      "import distutils.sysconfig as cg; print(cg.get_python_lib(plat_specific=True, standard_lib=False))"
```

Comment:
Triggers a warning in python3.10, see #924.

Reply:
@NicoRenaud we need to fix this, from [PEP 632](https://peps.python.org/pep-0632/)
```
Code that imports distutils will no longer work from Python 3.12.
```

Reply:
The code we need to replace is the following:
```python
import distutils.sysconfig as cg
print(cg.get_python_lib(1,0,prefix='${CMAKE_INSTALL_PREFIX}'))
```

Reply:
https://discourse.cmake.org/t/findpython3-how-to-specify-local-installation-directory-for-python-module/3580

Reply:
I guess this would be semi-reasonable:
```cmake
if (NOT CMAKE_INSTALL_PYTHON_LIBDIR)
  set(CMAKE_INSTALL_PYTHON_LIBDIR
    "${CMAKE_INSTALL_LIBDIR}/python${major}.${minor}/site-packages")
endif ()
```